webui/react/src/types.ts

```diff
@@ -707,3 +697,24 @@ export interface PriorityScheduler {
   defaultPriority: number;
   preemption: boolean;
 }
+
+/* Jobs */
+
+export interface Job extends Api.V1Job {
+  summary: Api.V1JobSummary;
+}
+export const JobType = Api.Determinedjobv1Type;
+export type JobType = Api.Determinedjobv1Type;
+export const JobState = Api.Determinedjobv1State;
+export type JobState = Api.Determinedjobv1State;
```

Comment:
**question:** does this work ok? is `const` needed?

Reply:
it works ok. without the `const` keyword, it interests it as a type and not value. if the enum from the generated code had a better/shorter name I'd use that but for now this seems cleaner. wdy think?